### DISCUSSION TITLE
Can I manually trigger a snippet to expand?

### DISCUSSION THREAD

**Discussion by tiagovla** [L3MON4D3/LuaSnip#657] [Q&A] [answered]
Let's say I have a snippet `s`. Can I programmatically expand it with something like `s:expand`? Do I need to register it anywhere?

**Comment by L3MON4D3** [comment, marked as answer]
Yes, and no need to register it, use `ls.snip_expand`, doc [here](https://github.com/L3MON4D3/LuaSnip/blob/master/DOC.md#api-reference)
